content/en/docs/release-notes/release-notes-0.14.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Release Notes"
+linkTitle: "v0.14"
+weight: 140
+type: "docs"
+---
+
+In `v0.14` we've focused on a number of areas:
+
+## Webhook changes
+
+The webhook component has now been made required, meaning any users that previously used the `no-webhook` variant will
+need to upgrade their installations to enable the webhook. This will be done automatically upon upgrading to the `v0.14`
+release, so no additional action is required.
+
+If you have issues running the webhook in your environment, we'd like to hear from you! We are aware of issues relating
+to firewall rules from the Kubernetes API server to the webhook pod(s) - we would like to gather together a corpus of
+configuration snippets that can be used to ensure the webhook is successfully deployed in these environments too.
+
+This change is required in order to support the upcoming changes to our API versions, as we introduce `v1alpha3`,
+`v1beta1` and `v1` over the coming months!
+
+## Improving our deployment and release process
+
+After reports of various issues installing on older Kubernetes and OpenShift versions, we've taken some time to revise
+our installation manifests.
+
+There are now two 'variants' to choose from, 'standard' and the 'legacy', with a simple way to know which to use:
+
+| Environment          | Variant to use             |
+| -------------------- | -------------------------- |
+| Kubernetes 1.15+     | `cert-manager.yaml`        |
+| OpenShift 4          | `cert-manager.yaml`        |
+| Kubernetes 1.11-1.14 | `cert-manager-legacy.yaml` |
+| OpenShift 3.11       | `cert-manager-legacy.yaml` |
+
+Please be sure to read the upgrade guide for more information on how to upgrade from a previous release.
```

Comment:
Can we note the restrictions on legacy?

Reply:
As in, what it enables/disables? That would be good, although I'd prefer to link to a table/section in the installation guide that outlines this instead of documenting it solely in release notes (where it will be lost to time). We could then link to it from here?

Reply:
Yeah a link would be nice at least some indication for people to be able to know there are limitations in this version